jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -177,10 +179,14 @@
    */
   private JaegerTracer tracer;
 
-  public Configuration(String serviceName) {
+  public Configuration(String serviceName, TracingFactory tracingFactory) {
```

Comment:
did you see my comment? https://github.com/jaegertracing/jaeger-client-java/pull/509#discussion_r208400650 ?

Reply:
I did. I'll comment more there. Need the tracing factory to construct the `JaegerTracer.Builder` subclass used in the config to create the tracer.